doc/jenkins-guide.md

```diff
@@ -62,10 +62,10 @@ be run on the worker itself, after SSH-ing in and switching to the
 
 ##### Out of memory
 
-First, get statistics on running processes for the machine: `ps aux | grep node | grep -v -egrep -ejava`.
+First, get statistics on running processes for the machine: `ps aux | grep node`.
 
```

Comment:
just wondering why the java one does not make sense anymore.  The jenkins agents still runs with Java.

Reply:
IMHO getting the stats of the `java` proc (or if there is none, or more then 1) is interesting. Also this is a simpler line...

Reply:
Ok I see now, that its a -v, so its adding not removing the java processes.  But in that case does removing the -egrep make sense?

Reply:
We should hit those procs with the `node` in `ci.nodejs.org`, e.g.:
```
iojs       839     1  1 647319 174936 0 /usr/bin/java -Xmx128m -jar /home/iojs/slave.jar -jnlpUrl https://ci.nodejs.org
                                                                                                             ^^^^
```